developer/src/kmc-package/test/test-package-compiler.ts

```diff
@@ -61,26 +91,27 @@ describe('KmpCompiler', function () {
 
     let kmpJson = null;
     assert.doesNotThrow(() => {
-      kmpJson = kmpCompiler.transformKpsToKmpObject(kpsPath, source);
+      kmpJson = kmpCompiler.transformKpsToKmpObject(source, kpsPath);
     });
 
     const kmpData = await kmpCompiler.buildKmpFile(kpsPath, kmpJson);
 
     let jszip = await zip.loadAsync(kmpData);
+    assert.isNotNull(jszip.file('kmp.json')); // kmp.json should be present
     // kmp file should contain the following files
     const expectedFiles = [
       'FONTLOG.txt', 'image002.png', 'KAK_Documentation_EN.pdf', 'KAK_Documentation_KH.pdf',
       'keyboard_layout.png', 'khmer_angkor.js', 'khmer_angkor.kmx', 'khmer_angkor.kvk',
       'khmer_busra_kbd.ttf', 'Mondulkiri-R.ttf', 'OFL.txt', 'OFL-FAQ.txt', 'readme.htm',
-      'splash.gif', 'welcome.htm'
+      'splash.gif', 'welcome.htm',
+      'kmp.json', // this seems to be included as well.
```

Comment:
@mcdurdin this file seems to get automatially added

Reply:
Yes, this makes line 100 above redundant, as we are now checking all in one array.

```suggestion
      'kmp.json', // standard .kmp metadata file
```